### ISSUE TITLE
Wrap scipy.sparse ?

### ISSUE THREAD

**Issue by tdegeus** [xtensor-stack/xtensor-sparse#55]
It's been a while that I followed this. I wonder, it there a support to wrap scipy sparse matrices, like there is in Eigen : https://pybind11-jagerman.readthedocs.io/en/stable/advanced.html#transparent-conversion-of-dense-and-sparse-eigen-data-types ?

**Comment by stellarpower** [comment]
I believe from the docs, a copy is required in these cases - but I think this is also happening for dense numpy arrays with xtensor-python, and, it's a start if nothing else. I think acceptable if it's a known caveat that's highlighted.

**Comment by JohanMabille** [comment]
>  I think this is also happening for dense numpy arrays with xtensor-python

No, xtensor-python can operate on numpy arrays in place. The idea of this repo is not to depend on a python library (although an upstream package could provide bindings to python), and is not limited to 2D.
